ledgerjs-fio/example-node/index.ts

```diff
@@ -0,0 +1,88 @@
+import TransportNodeHid from "@ledgerhq/hw-transport-node-hid";
+import { Fio, HARDENED, GetPublicKeyRequest, SignTransactionRequest, Transaction } from "../src/fio";
+const fetch = require('node-fetch')
+const readline = require('readline');
```

Comment:
These should be converted to `import` statements for consistency.
```suggestion
import fetch from 'node-fetch';
import readline from 'readline';
```
And you never use `readline` here so in fact it should be:
```suggestion
import fetch from 'node-fetch';
```